src/mmw/js/src/data_catalog/templates/form.html

```diff
@@ -1,6 +1,35 @@
-<i class="data-catalog-search-icon fa fa-search"></i>
-<input
-    class="data-catalog-search-input"
-    type="text"
-    placeholder="Search data sources"
-/>
+<button class="btn btn-link date-filter-toggle small">{{filterText}}</button>
+
+{% if showingFilters %}
+<div class="data-catalog-search-dates">
+    <div class="form-group">
+        <label>Before</label>
+        <input
+            class="data-catalog-date-input form-control"
+            data-date-range="start"
+            value="{{dateStart}}"
```

Comment:
The current setup is wrong:

![image](https://user-images.githubusercontent.com/1430060/28581149-880b7e82-712f-11e7-9728-67b8e1768418.png)

since this describes an infinite date range with a gap in between. However, reading "After" come first and "Before" later could also be counterintuitive. Perhaps we should use other labels, like "From" and "To" or "Till".